crates/reliable-broadcast/src/lib.rs

```diff
@@ -148,7 +148,7 @@ where
                                 }
                             },
                             Err(e) => {
-                                info!(error = ?e, "rpc to {} failed", receiver);
+                                debug!(error = ?e, "rpc to {} failed", receiver);
```

Comment:
I prefer sample! with warn! for this, @danielxiangzl @zekun000 WDYT?

Reply:
+1 to make this warn! with sample for prod, but can we check if debug log is enabled and print this in debug without sampling? That way, we can retain this behavior for forge debugging.

Reply:
makes sense, this can go with sampling since there's retries for it

Reply:
SGTM 😄 Couldn't find an easy way to detect if debug logs are enabled, so was lazy and just added two log lines (debug logs will get dropped in production).